src/runtime/pkg/katautils/config.go

```diff
@@ -1278,6 +1282,7 @@ func GetDefaultHypervisorConfig() vc.HypervisorConfig {
 		EnableIOThreads:          defaultEnableIOThreads,
 		Msize9p:                  defaultMsize9p,
 		HotplugVFIOOnRootBus:     defaultHotplugVFIOOnRootBus,
+		ColdPlugVFIO:             defaultColdPlugVFIO,
```

Comment:
when is this value consumed? is it used when we use an older config.toml, which doesn't have any value defined for `cold_plug_vfio`?

Reply:
Older or newer config, if `cold_plug_vfio` is commented out, which it should be per default , the defaultColdPlugVFIO is set to "no-port".